buildscripts/dockertest.py

```diff
@@ -312,13 +399,16 @@ def test_build(fout, image, docker_version):
     clear_tuf()
     # build
     # simple dockerfile to test building with trust
-    dockerfile = "FROM {0}:{1}\nRUN sh\n".format(image, docker_version)
+    dockerfile = "FROM {image}:{tag}\nRUN sh\n".format(
```

Comment:
yeah I kept messing up the 0, 1, 2, 3 :)  hopefully this will make it easier for me to keep straight